### PR TITLE
Add is_floating_point to docs

### LINKED ISSUE #15700: [docs] Document torch.is_floating_point
Reported from https://discuss.pytorch.org/t/is-floating-point-method-property/33677

### PR DESCRIPTION
**Pull request by vishwakftw**
Fixes #15700 .

Changelog:

- Expose torch.*.is_floating_point to docs

cc: @zou3519

### CODE REVIEW

File: test/test_torch.py [hunk 1]
```diff
@@ -197,7 +197,6 @@ def test_namespace(ns, *skips):
                        'index_put',
                        'is_coalesced',
                        'is_distributed',
```
Comment (zou3519) [inline test/test_torch.py]: is this a list of all docs that we don't have?

**Review by zou3519 — APPROVED** [review]
thanks!

File: test/test_torch.py (hunk 1 above)
Comment (vishwakftw) [reply test/test_torch.py]: I suppose so. I haven't checked this.

**Comment by vishwakftw** [comment]
@zou3519 This doesn't seem to have been merged in, are there any issues that I need to get fixed?

**Comment by zou3519** [comment]
Let's try that again...

**Comment by zou3519** [comment]
Merged by commit 95febdf

**Pull request closed without merging**